libs/punchapp/src/lib/config/tableConfig.tsx

```diff
@@ -1,11 +1,30 @@
-import { DateCell, DescriptionCell } from '@cc-components/shared';
+import {
+  DateCell,
+  DescriptionCell,
+  hasProperty,
+  LinkCell,
+  proCoSysUrls,
+  StatusCell,
+  StatusCircle,
+  statusColorMap,
+} from '@cc-components/shared';
 import { GridConfig, ICellRendererProps } from '@equinor/workspace-fusion/grid';
 import { Punch } from '../types';
 export const tableConfig: GridConfig<Punch> = {
   columnDefinitions: [
     {
       field: 'Punch',
       valueGetter: (pkg) => pkg.data?.punchItemNo,
+      valueFormatter: (pkg) =>
+        pkg.data?.commissioningPackageId
+          ? proCoSysUrls.getCommPkgUrl(pkg.data.commissioningPackageId)

```

Comment:
```suggestion
          ? proCoSysUrls.getPunchUrl(pkg.data.punchItemNo)
```

Don't use commissioningPackageId to generate a punch url, use punchItemNo instead